### ISSUE TITLE
flamer is now unusable

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#2088]
Since the last nerfs of flamer, the weapon became completely unusable in practice.

**Comment by Gireen** [comment]
what kind of bug is "unusable" ?
are you not able to buy it?

**Comment by bmorel** [comment]
Right. It is technically usable, so this should go in gameplay. The problem is, it's now a worthless weapon.
